helm-chart/scc-admin-frontend/templates/deploy-secret-serviceaccount.yaml

```
@@ -0,0 +1,34 @@
+    verbs: ["create", "patch", "get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: create-secrets-to-{{ include "scc-admin-frontend.serviceAccountName" . }}-deploy-secret
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "10"
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "scc-admin-frontend.serviceAccountName" . }}-deploy-secret
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)